docs/source/constraints.rst

```diff
@@ -31,38 +31,26 @@ AASd-005    A revision requires a version.      ✅
             This means, if there is no
             version there is no
             revision either.
-AASd-006    if both, the value and the valueId  ❌     Uncheckable, cannot check the value
-            of a Qualifier are                         of what value_id points to
-
-            present then the value needs to
-            be identical to the value of
-
-            the referenced coded
+AASd-006    If both, the value and the valueId
+            of a Qualifier are present,
+            the value needs to be identical to
+            the value of the referenced coded
```

Comment:
I think this change here should be reverted, unless the constraint is implemented now.

Reply:
You are right, the text should only be reformulated (if -> If). I restored the lost content of `WIP` and `See`
see: [commit](https://github.com/eclipse-basyx/basyx-python-sdk/pull/85/commits/f8e93314151ef055080da492e7153b0849a09ed3)